package.json

```diff
@@ -9,9 +9,10 @@
     "format:lint-client": "eslint --ext js,jsx -c ./client/.eslintrc ./client --fix",
     "format:lint-server": "eslint --ext js,jsx -c ./server/.eslintrc ./server --fix",
     "update-all": "run-s 'update:*'",
+    "update-apps": "run-p 'update:client' 'update:server'",
     "update:root": "npm i",
-    "update:client": "npm i --prefix client",
-    "update:server": "npm i --prefix server",
+    "update:client": "npm ci --prefix client",
```

Comment:
`npm ci` will be slower for local development, we should use `npm i` here

Reply:
npm i is updating package-lock.json .....

Reply:
then package-lock is out of date for some reason 🤔

Reply:
yes...thats true

Reply:
Reverted.